src/components/DocumentList/DocumentList.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { ReactElement } from "react";
+import { ScrollView } from "react-native";
+import { DocumentListItem } from "./DocumentListItem";
+
+export interface DocumentItem {
+  id: string;
+  title: string;
+  isVerified?: boolean;
+  lastVerification?: number;
+}
+
+export interface DocumentList {
+  documents: DocumentItem[];
+  navigateToDoc: (documentId: string) => void;
+}
+
+export const DocumentList = ({
+  documents,
+  navigateToDoc
+}: DocumentList): ReactElement => {
```

Comment:
We can utilise `FunctionComponent<DocumentList>` then we don't need to explicitly define the return type.

Reply:
![image](https://user-images.githubusercontent.com/7406870/68914826-c3d38a00-079a-11ea-8d43-979c39cf1c41.png)

Problem is that it is asking for props validation now that the return type has been removed.